src/content/eth2/shard-chains/index.md

```diff
@@ -86,15 +90,17 @@ The Eth2 upgrades are all somewhat interrelated. So let’s recap how the shard
 
 ### Shards and the beacon chain {#shards-and-beacon-chain}
 
-The beacon chain contains all the logic for keeping shards secure and synced up. The beacon chain will coordinate the stakers in the network, assigning them to shards they need to work on. And it will also facilitate communication between shards by receiving and storing shard transaction data that is accessible by other shards. This will give shards a snapshot of Ethereum’s state to keep everything up-to-date.
+The Beacon Chain contains all the logic for keeping shards secure and synced up. The Beacon Chain will coordinate the stakers in the network, assigning them to shards they need to work on. And it will also facilitate communication between shards by receiving and storing shard transaction data that is accessible by other shards. This will give shards a snapshot of Ethereum’s state to keep everything up-to-date.
 
 <ButtonLink to="/eth2/beacon-chain/">The Beacon Chain</ButtonLink>
 
-### Shards and the docking {#shards-and-docking}
+### Shards and the merge {#shards-and-docking}
+
+Originally, plans were made to implement shard chains prior to the incorporation of the Ethereum mainnet as a primary means of scaling. As research and development of layer 2 scaling solutions have boomed, the priority of core Ethereum devs shifted to expediting the switch a fully proof-of-stake driven mainnet. This means that the remaining shard chains will not be complete prior to switching mainnet to proof-of-stake.
```

Comment:
we can't just repeat content across pages – we'll get penalised by Google. This paragraph should talk about how shard chains will build off of a mainnet powered by layer 2 solutions, supercharging the scalability. Also as the whole system will be proof-of-stake already, shard chains will fit seamlessly rather than having to navigate a world where we had a PoW mainnet with a PoS beacon chaiin.

Reply:
Re: the update. What do we mean by fertile? I'm not sure this is clear still?